themes/stanford_basic/decoupled-menu/src/main-menu.island.tsx

```diff
@@ -132,12 +132,21 @@ const SearchContainer = styled.div`
 export const MainMenu = ({}) => {
   useWebComponentEvents(islandName)
   const [menuItems, setMenuItems] = useState<MenuContentItem[]>(window.drupalSettings?.stanford_basic?.decoupledMenuItems || []);
+  const [utilityNavLinks, setNavLinks] = useState([]);
   const [menuOpen, setMenuOpen] = useState<boolean>(false);
   const buttonRef = useRef<HTMLButtonElement | null>(null);
   const navRef = useRef<HTMLElement | null>(null);
   useOutsideClick(navRef, () => setMenuOpen(false));
 
   useEffect(() => {
+    const links = document.getElementsByClassName('utility-nav')
+    if (links) {
+      const utilityObj = Array.from(links).map(link => ({
```

Comment:
@pookmish I couldn't use `.map` without the `Array.from` because the links came back as and HTMLCollection.  Can you point out why it doesn't work?